i18n/en.json

```diff
@@ -4624,35 +4100,35 @@
   },
   {
     "id": "model.emoji.user_id.app_error",
-    "translation": ""
+    "translation": "Invalid creator id"
   },
   {
     "id": "model.file_info.get.gif.app_error",
     "translation": "Could not decode gif."
   },
   {
     "id": "model.file_info.is_valid.create_at.app_error",
-    "translation": ""
+    "translation": "Invalid value for create_at."
```

Comment:
for which query/API call? (similarly for below)

Reply:
Is this when uploading a file attachment?

Reply:
And where can I find the accepted values, are they in our API docs?

Reply:
It is a validation for any FileInfo used in the entire app, but mainly related to post attachments and user/team images (I think). And the CreateAt field is valid if has a value > 0